frontend/pages/index/art.vue

```diff
@@ -109,7 +108,7 @@
           </BadgeFilter>
 
           <!-- Grant Badge Filter -->
```

Comment:
I've seen that this was commented. If it's something you'll use in the future, please note that. Otherwise, other team members might delete this block.

It's noted in the ops as well: https://github.com/countable-web/ops-manual/blob/master/developers/CODING_STANDARDS.rst#comments